### DISCUSSION TITLE
How do I activate the Graph Analysis view?

### DISCUSSION THREAD

**Discussion by nascifMode** [SkepticMystic/graph-analysis#24] [Q&A] [answered]
It feels like I am missing something basic, but I can't find how to open the Graph Analysis view.

I installed and enabled the plugin, but there is no corresponding option in the "More options" menu and the only related command that appears in the Command Palette is "Graph Analysis: Refresh Graph Analysis View" which appears to work (it displays a "Index refreshed" pop-up) but does not show anything. 
There is also no new launcher icon in the toolbar.
I am on Windows, Obsidian v0.12.19 (up-to-date), Graph Analysis v0.2.7

**Comment by HEmile** [comment, marked as answer]
The view should appear on the panel on the right side here, when loading the plugin: ![](https://i.imgur.com/4ZUXfWt.png) . Note the selected pane, where the icon looks like a small graph with 5 nodes.

**Reply by nascifMode** [reply]
But how do I cause the panel to appear in the first place? There is no new entry on the options panel, toolbar or command pallet
![menu_options](https://user-images.githubusercontent.com/82056060/140341582-8c03be9b-8219-4699-9ba7-a2f44320902d.png)
.

**Reply by HEmile** [reply]
Are you sure you looked at the right sidebar? https://help.obsidian.md/User+interface/Workspace/Sidebar
The sidebar can be expanded by clicking on the vertical bar on the very right of your screen: 
![image](https://user-images.githubusercontent.com/1289165/140375465-739ab9f9-73d5-4197-a046-b58fcef87ed5.png)

**Reply by TookiTheGreat** [reply]
I think I may have at least part of an answer: I couldn't find an "Open Graph Analysis View" command either, nor did I see the icon in the panel. I did have several other icons in the panel from other plugins I had installed. Once I removed those plugins, the Graph Analysis icon showed up and I could open the analysis view as expected. I still don't have an "Open Graph Analysis View" command, though.
So, it appears that the side-panel only has limited space for extra icons and the ones that don't fit just disappear in limbo. Try removing some of the plugins that place icons on the side-panel until you see the graph analysis icon appear.
For the record: I'm running Obsidian 0.12.19 on an iMac M1 running Monterey.

**Reply by nascifMode** [reply]
Emile got it, I was not aware I had a collapsed right sidebar. So, I was indeed missing something basic!
Thanks both!

**Reply by SkepticMystic** [reply]
Good to hear! Let us know if you've got any other questions or want to share some feedback :)
